incubator/kubevirt/README.md

```diff
@@ -0,0 +1,61 @@
+# KubeVirt Helm Chart
+It will deploy [KubeVirt](https://github.com/kubevirt/kubevirt) resources to make possible to create VM's as part of k8s environment.
+You can find more information about VM yaml specification under [KubeVirt User Guide](https://kubevirt.gitbooks.io/user-guide/)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+```
+
+You will need to give additional permissions to the user `system:serviceaccount:kube-system:default`, it will be valid only for cases when you initiated `helm` under `kube-system` namespace.
+
+```
+$ kubectl create clusterrolebinding --user system:serviceaccount:kube-system:default kube-system-cluster-admin --clusterrole cluster-admin
+```
+
+You will need to specify your master IP under `master.ip` variable and master hostname under `master.selector` variable, because most components must run on the same node and will talk via external IP(I hope it will change soon).
```

Comment:
KubeVirt compoinents don't talk via services with each other. They don't need external IPs. You only need a service for the user which can use the external IP to access the componenets from outside the cluster.

Reply:
Regarding the master. selector requriement. We have in our development environment in KubeVirt a selector, but that is not needed normally. That is just for our test environment.